dashboard/src/components/SideMenu/SendFeedback.tsx

```diff
@@ -0,0 +1,74 @@
+import { MdOutlineFeedback } from 'react-icons/md';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from '@/components/ui/alert-dialog';
+
+import {
+  FEEDBACK_ISSUE_URL,
+  FEEDBACK_EMAIL_TO,
+} from '@/utils/constants/general';
+
+import { NavigationMenuItem } from '@/components/ui/navigation-menu';
+
+import NavLink from './NavLink';
+
+const get_encoded_url = (): string => encodeURIComponent(window.location.href);
+
+const onGitHubClick = (): void => {
+  window.open(`${FEEDBACK_ISSUE_URL}&body=${get_encoded_url()}`, '_blank');
```

Comment:
let's create a body with a section `======== Don't Change======== ` where we will put the context, something like that